File: 3731-find-missing-elements/3731-find-missing-elements.cpp
class Solution {
public:
    vector<int> findMissingElements(vector<int>& nums) {
        vector<int> v;
        int mn = INT_MAX;
        int mx = -INT_MAX;
        int arr[101] = {0,};

        for(int num : nums){
            arr[num] = 1;
            mn = min(mn, num);
            mx = max(mx, num);
        }
        for(int i=mn;i<mx;i++){
            if(!arr[i]) v.push_back(i);
        }
        return v;
    }
};

File: 3731-find-missing-elements/3731-find-missing-elements.java
class Solution {
    public List<Integer> findMissingElements(int[] nums) {
        Arrays.sort(nums);
        List<Integer> list = new ArrayList<>();
        int index = 0;
        for(int i=nums[0];i<nums[nums.length-1];i++){
            if(nums[index] == i) index++;
            else list.add(i);
        }
        return list;
    }
}